### ISSUE TITLE
Unable to get Canada, UK stocks information

### ISSUE THREAD

**Issue by 99coder99** [ranaroussi/yfinance#357]
Thanks for the yfinance cool-tool @ranaroussi 

Two quick questions, if you can provide some info please?

1) While the `msft.info` works perfectly for US markets, it is not working for UK and Canadian stocks.

For example, for UK stocks like Vodafone:

```
import yfinance as yf

vod = yf.Ticker("VOD.L")

# get stock info
vod.info
```
and similarly for Canadian stocks listed on Toronto Stock Exch, 
```
import yfinance as yf

aly = yf.Ticker("ALY.TO")

# get stock info
aly.info
```
is returning the error: "`IndexError: list index out of range`"

Is there any error in the code that I am using? Kindly let me know.

2) Is it possible to save the output to a text file? I was unable to locate the command for that. Tried the standard to_csv command `msft.to_csv('msft.csv')`, but it is throwing an error.

3) Can the .info be run on multiple tickers in one go? 

Your inputs will be very useful. Kindly let me know, and thanks again for the great utility!

**Comment by rodrigobercini** [comment]
1) `While the msft.info works perfectly for US markets, it is not working for UK and Canadian stocks.`
Try applying the fixes described in PR #179.
Just be aware that that fix might return inconsistent data for non-US tickers. Be sure to validate the data fetched.

2) ` Is it possible to save the output to a text file`
You must convert msft.info into a data frame before saving as csv.

```
import pandas as pd
import yfinance as yf
ticker = yf.Ticker('AAPL')
pd.DataFrame(ticker.info.items())
```

3) `Can the .info be run on multiple tickers in one go?`
This one I'm not sure. One can fetch multiple tickers in one request. But I guess one has to go through a for loop to consolidate the data into a single dataframe.

```
tickers = yf.Tickers('MSFT AAPL GOOG')
tickers.tickers.MSFT.info
tickers.tickers.AAPL.info
tickers.tickers.GOOG.info
```

**Comment by Jay-Plumb** [comment]
A slight tangent to the above but I noticed that ticker symbol `WFC` throws an  `IndexError: list index out of range` when calling `yf.Ticker("WFC").info`.
